tests/CoreFX/CoreFX.issues.json

```diff
@@ -739,6 +739,32 @@
             ]
         }
     },
+    {
+        "name": "System.IO.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.IO.Tests.CloseTests.AfterDisposeThrows",
```

Comment:
Is it expected that these tests fail in the testing that use the latest version of CoreFX? 
[Example](https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/arm64_cross_checked_ubuntu16.04_corefx_baseline_tst/159/consoleFull#11172936692d31e50d-1517-49fc-92b3-2ca637122019):
```

Could you please create an issue and link it to these exclusion  to track when we need to reenable them?

    System.IO.Tests.CloseTests.AfterDisposeThrows [FAIL]
04:18:43       Assert.Null() Failure
04:18:43       Expected: (null)
04:18:43       Actual:   MemoryStream { CanRead = False, CanSeek = False, CanTimeout = False, CanWrite = False, Capacity = (throws ObjectDisposedException), ... }
04:18:43       Stack Trace:
04:18:43         /mnt/j/workspace/dotnet_coreclr/master/jitstress/arm64_cross_checked_ubuntu16.04_corefx_baseline/_/fx/src/System.IO/tests/StreamWriter/StreamWriter.CloseTests.cs(44,0): at System.IO.Tests.CloseTests.ValidateDisposedExceptions(StreamWriter sw)
```

Reply:
Yes, these types of failures are expected in the current jitstress setup.

This problem will be fixed once https://github.com/dotnet/coreclr/pull/23388 goes through and jitstress runs are switched to use it. Also, as part of that work all exclusions in CoreFX.issues.json will be deleted - no need to track them individually.